Files/seller/routes.py

```diff
@@ -63,9 +54,11 @@ def get_users():
     response = result
     return response
 
-@seller.get('/<int:user_id>')
+@seller.get('/')
```

Comment:
Route for **get_users** and **get_user_byID** is same. How will it get differentiated?